### DISCUSSION TITLE
Maybe, Something wrong with the starter

### DISCUSSION THREAD

**Discussion by No54** [bcakmakoglu/vue-flow#257] [Q&A] [answered]
I create a vite vue test project 

```pnpm create vite@latest```

then install vue-flow,

I copy the codes from the document, but it seems something wrong... 

Would you please give some help ? Thanks!

![image](https://user-images.githubusercontent.com/38535403/181917901-553e9479-a1ac-406c-8626-b9f268025c53.png)

**Comment by bcakmakoglu** [comment, marked as answer]
You're missing the styles.

1. Your Vue Flow container needs to have a height.
2. You need to import the necessary Vue Flow styles. (https://vueflow.dev/guide/theming.html)

**Reply by No54** [reply]
It work! Thank you very much for your help! ^_^
